magento/magento2
# xssFiltrationPattern boundary script tag restriction

Add script boundary as word

If you have an html like:

```
<body>
    <p class="product-description">
     hello world product description
    </p>
</body>

```

The problem is the class: product-de**script**ion



### Manual testing scenarios (*)

1. Create a CMS Page with content:
```
<p class="product-description">
     hello world product description
    </p>
```
2. Save and see error: 
<img width="643" alt="image" src="https://github.com/user-attachments/assets/1e7273b9-3b1b-430e-9d35-d2327cd1c561">


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39384: xssFiltrationPattern boundary script tag restriction
